untraced/yksg/src/lib.rs

```diff
@@ -203,7 +203,7 @@ struct StackFrame {
 /// over.
 pub struct StopgapInterpreter {
     /// Active stack frames (most recent last).
-    frames: Vec<StackFrame>,
+    frames: Vec<Frame>,
```

Comment:
I think the reasoning was that from the StopgapInterpreter's point of view `frames` is the stack on which is stores the stack frames, but I agree with this change nonetheless.